src/main/java/com/javacode/entities/Comment.java

```diff
@@ -0,0 +1,44 @@
+package com.javacode.entities;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "comments")
+@Setter
+@Getter
+@NoArgsConstructor
+
+public class Comment {
+	@Id
+	@Column(name = "id")
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int id;
+
+	@Column(name = "content")
+	private String content;
+
+	@Column(name = "reply_id")
+	private int reply_id;
+
+	@ManyToOne
+	@JoinColumn(name = "user_id", insertable = false, updatable = false)
+	private User user;
+	
+	@OneToOne(cascade = CascadeType.ALL)
```

Comment:
Sao lại là quan hệ `one to one ` em
phải là ` many to one ` chứ - tức là 1 job có nhìu comment chú ơi